style/js/shield_defs.js

```diff
@@ -2279,27 +2280,44 @@ export function loadShields(shieldImages) {
     2,
     1
   );
-
   [
-    "Amsterdam",
-    "Den Haag",
-    "Rotterdam",
-    "Nijmegen",
-    "Parkstad",
-    "Zaanstad",
+    "NL:S:Amsterdam",
+    "NL:S:Den Haag",
+    "NL:S:Nijmegen",
+    "NL:S:Parkstad",
+    "NL:S:Rotterdam",
+    "NL:S:Zaanstad",
+    "NL:binnenstedelijke_ring", // also for Curacao
   ].forEach(
-    (city) =>
-      (shields["NL:S:" + city] = {
+    (network) =>
+      (shields[network] = {
         backgroundImage: shieldImages.shield40_nl_city,
         textColor: Color.shields.black,
         padding: {
           left: 3,
           right: 3,
-          top: 4,
-          bottom: 5,
+          top: 3,
+          bottom: 3,
         },
       })
   );
+  [
+    "NL:R:Ommen",
+    "NL:R:Schouwen",
+    "NL:R:Sluis",
+    "NL:R:Spaarnwoude",
+    "NL:R:Voorthuizen",
+    "NL:R:IJmuiden",
+  ].forEach(
+    (network) =>
+      (shields[network] = roundedRectShield(
```

Comment:
It would be a bit cleaner code-wise to extract the NL:R prefix from above say:
```
(city) =>
(shields[`NL:R:${city}`] = ...
```

This would make it clear in the code that this is a list of city networks with a common prefix.

Reply:
Added new commit to do this (with "place" instead of "city", some are simply rural areas).